Projects/calc.py

```diff
@@ -1,22 +1,47 @@
-ch = int(input("1: Add \n2: Subtract \n3: Multiply \n4: Divide \n5: Square root \nEnter your choice: "))
-if ch == 1:
-    num1 = float(input("Enter number: "))
-    num2 = float(input("Enter number: "))
-    print("Addition: " + str(num1 + num2))
-elif ch == 2:
-    num1 = float(input("Enter first number: "))
-    num2 = float(input("Enter second number: "))
-    print("Subtraction: " + str(num1 - num2))
-elif ch == 3:
-    num1 = float(input("Enter number: "))
-    num2 = float(input("Enter number: "))
-    print("Multiplication: " + str(num1 * num2))
-elif ch == 4:
-    num1 = float(input("Enter number: "))
-    num2 = float(input("Enter number: "))
-    print("Division: " + str(num1/num2))
-elif ch == 5:
-    num1 = float(input("Enter number: "))
-    print("Square root: " + str(num1**(1/2))) 
-else:
-    print("Invalid input!!")
\ No newline at end of file
+def calculator():
+    print("Options:\n\t[1] Add \n\t[2] Subtract \n\t[3] Multiply \n\t[4] Divide \n\t[5] Power \n\t[6] Square root")
+    ch = int(input("\t--> "))
+    
+    if ch == 1: #addition
+        num1 = float(input("1st number --> "))
+        num2 = float(input("2nd number --> "))
+        print(f"{num1} + {num2} = {num1 + num2}")
+    elif ch == 2: #subtraction
+        num1 = float(input("1st number --> "))
+        num2 = float(input("2nd number --> "))
+        print(f"{num1} - {num2} = {num1 - num2}")
+    elif ch == 3: #multiplication
+        num1 = float(input("1st number --> "))
+        num2 = float(input("2nd number --> "))
+        print(f"{num1} x {num2} = {num1 * num2}")
+    elif ch == 4: #division
+        num1 = float(input("Dividend --> "))
+        num2 = float(input("Divisor  --> "))
```

Comment:
@KennyOliver 
Could you please add one more correction? Dividing 0 by 0 should not be considered valid. So make the program display an error message when the user enters 0 as the divisor.

Reply:
Sure, I’ve just made a new PR.